### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by mahammad-mostafa**
- Created nested resources routes for `users` & `posts`
- Built an empty controller file `users_controller` & `posts_controller`
- Included placeholder view file for both `users` & `posts`

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class UsersController < ApplicationController
+end
```
Comment (Munsa1) [inline app/controllers/users_controller.rb:2]: - [ ] Kindly ensure your controller has #index and #show actions. It's okay for the actions to be empty for now

e.g
```ruby
class UsersController < ApplicationController
  
  def index; end

  def show; end

end
```

File: app/controllers/posts_controller.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class PostsController < ApplicationController
+end
```
Comment (Munsa1) [inline app/controllers/posts_controller.rb:2]: - [ ] Kindly ensure your controller has #index and #show actions. It's okay for the actions to be empty for now

e.g
```ruby
class PostsController < ApplicationController
  
  def index; end

  def show; end

end
```

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @mahammad-mostafa, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Protected Unecessary Endpoints :heavy_check_mark: 
Correct Git Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ 🎉 🎉 ✔️ 

Hi @mahammad-mostafa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] All changes are made ✔️ 
- [ ] No linters errors ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**